src/components/common/Checkbox/Checkbox.tsx

```diff
@@ -9,16 +9,18 @@ type Props = JSX.IntrinsicElements['button'] & {
 
 export const Checkbox: FC<Props> = ({ isChecked, setIsChecked, ...attr }) => {
   const classNames =
-    'size-18 rounded-4 border-2 flex items-center justify-center cursor-pointer';
-  const borderColor = isChecked
-    ? 'border-primary bg-primary'
-    : 'border-white/60';
+    'size-18 rounded-4 flex items-center justify-center cursor-pointer';
+  const borderBgColor = isChecked
+    ? 'bg-gradient-to-r from-primaryGradient-first to-primaryGradient-middle'
```

Comment:
Would be worth creating a dedicated class for that.

Reply:
Done